From b33a6a83687fc58cd5f662d44ba8819498cb80a4 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Thu, 9 Aug 2018 14:10:32 +0800
Subject: [PATCH] socket6-perl: fix configure error

only do IPv6_CHECK_INET_NTOP when not cross compile to fix below
error: checking for working inet_ntop..., configure: error:
cannot run test program while cross compiling

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 configure.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 1fe22e1..1a7cf94 100644
--- a/configure.in
+++ b/configure.in
@@ -63,7 +63,9 @@ if test $ac_cv_lib_inet6_getaddrinfo = yes; then
 	INET6LIBS="-L$ipv6_cv_dir/lib -linet6"
 fi
 
-IPv6_CHECK_INET_NTOP()
+if test "$cross_compiling" != yes; then
+	IPv6_CHECK_INET_NTOP()
+fi
 IPv6_CHECK_SA_LEN()
 IPv6_CHECK_SIN6_SCOPE_ID()
 IPv6_CHECK_SOCKLEN_T()
-- 
2.7.4

